File: ympa_fsharp_giraffe_server/src/model/OfferMappingInfoDTO.fs
namespace ympa_fsharp_giraffe_server.Model

open System
open System.Collections.Generic
open ympa_fsharp_giraffe_server.Model.OfferMappingDTO

module OfferMappingInfoDTO =

  //#region OfferMappingInfoDTO


  type OfferMappingInfoDTO = {
    Mapping : OfferMappingDTO;
    AwaitingModerationMapping : OfferMappingDTO;
    RejectedMapping : OfferMappingDTO;
  }
  //#endregion

File: src/main/scala/org/openapitools/server/model/OfferMappingInfoDTO.scala
package org.openapitools.server.model


/**
 * Информация о карточке товара.
 *
 * @param mapping  for example: ''null''
 * @param awaitingModerationMapping  for example: ''null''
 * @param rejectedMapping  for example: ''null''
*/
final case class OfferMappingInfoDTO (
  mapping: Option[OfferMappingDTO] = None,
  awaitingModerationMapping: Option[OfferMappingDTO] = None,
  rejectedMapping: Option[OfferMappingDTO] = None
)

